corehq/apps/es/case_search.py

```diff
@@ -272,6 +272,26 @@ def case_property_missing(case_property_name):
     )
 
 
+def case_property_geo_distance(geopoint_property_name, lat, lon, distance):
+    """Filters cases to those within a certain distance of the provided lat/lon
+
+    Distance should be a string like "10km" or "3mi". For options, see:
```

Comment:
This'd cause an ES error like
```
corehq.apps.es.exceptions.ESError: TransportError(400, 'search_phase_execution_exception', 'For input string: "1000kl"')
```
If it's okay with you I'd like to include validation in the PR that makes use of this filter.  Since the user will be providing the distance in some format, that PR will need user-facing validation, and I thought it'd be easier to do both together.